pkg/registration/supply_chain_factory.go

```diff
@@ -330,6 +332,14 @@ func (f *SupplyChainFactory) buildClusterSupplyBuilder() (*proto.TemplateDTO, er
 	return clusterSupplyChainNodeBuilder.Create()
 }
 
+func (f *SupplyChainFactory) buildNodeGroupSupplyBuilder() (*proto.TemplateDTO, error) {
```

Comment:
Is this used somewhere? Wouldnt the usage of this mean that this will then be considered to be processed in supply chain?

Reply:
You're right. This is not needed now. Have removed  it